### PR TITLE
[JENKINS-22316] Show candidates based on input value

### PR DESCRIPTION
**Pull request by cupsadarius**
Show the dropdown candidates based on the inputed value in editable selects thus providing some sort of autocomplete functionality.

### CODE REVIEW

File: src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js [hunk 1]
```diff
@@ -50,12 +50,14 @@ Behaviour.register({"SELECT.staticCombobox": function(e) {
     orig.parentNode.removeChild(orig);
     
     /*
-     * Changes from the original 3:
-     *   Original behavior: Show candidates that matches the current incomplete input.
-     *   Changed behavior : Show all the candidates, not concerning with the current incomplete input.
+     * Changes from the original 4:
+     *   Original behavior: Show all the candidates, not concerning with the current incomplete input.
+     *   Changed behavior : Show candidates that matches the current incomplete input.
```
Comment (ikedam) [inline src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js]: The "original"means the combobox widget provided by Jenkins core.
https://github.com/jenkinsci/jenkins/blob/jenkins-2.89.3/core/src/main/resources/lib/form/combobox/combobox.js

The number of "Changes from the original" is an ordinal one, and you shouldn't change it.

**Comment by ikedam** [comment]
Link to the JIRA issue: https://issues.jenkins-ci.org/browse/JENKINS-22316

File: src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js (hunk 1 above)
Comment (cupsadarius) [reply src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js]: should i then revert the change number and update the commit?

**Comment by ikedam** [comment]
@cupsadarius 

Hmm...
I created #36 , updating the code comment so that it explains the changes from `<f:combobox>`.
Would you review that?

**Comment by ikedam** [comment]
Merged as #36.

**Comment by ikedam** [comment]
@cupsadarius  Sorry for having you wait for long time.
I released this change as extensible-choice-parameter-1.5.0.
It will be available in the update center in a day.

**Comment by cupsadarius** [comment]
@ikedam no problem! Thanks!

**Pull request closed without merging**